### DISCUSSION TITLE
e-katalog.lkpp.go.id/home

### DISCUSSION THREAD

**Discussion by 3xploiton3** [realodix/AdBlockID#1572] [Q&A] [answered]
Minta tolong, buatkan filter untuk block popup ini,
saya coba pakai filter dibawah ini, window jadi tidak bisa di scroll

> e-katalog.lkpp.go.id##.in.fade.modal
e-katalog.lkpp.go.id##.in.fade.modal-backdrop

![image](https://github.com/realodix/AdBlockID/assets/19517680/90b77529-897f-4d5f-8510-ab2cbfea239e)

**Comment by realodix** [comment, marked as answer]
```adblock
e-katalog.lkpp.go.id##.modal#home-notifikasi
e-katalog.lkpp.go.id##body > .modal-backdrop
e-katalog.lkpp.go.id##body:remove-class(modal-open)
```
